.cirrus.yml

```diff
@@ -103,6 +104,16 @@ build_task:
               # ID for re-use of build output
               CI_DESIRED_RUNTIME: crun
               CI_DESIRED_NETWORK: netavark
+              CI_DESIRED_DATABASE: boltdb
+        - env: &sqliteenvvars
+              DISTRO_NV: ${FEDORA_NAME}
+              # Not used here, is used in other tasks
+              VM_IMAGE_NAME: ${FEDORA_CACHE_IMAGE_NAME}
+              CTR_FQIN: ${FEDORA_CONTAINER_FQIN}
+              # ID for re-use of build output
+              CI_DESIRED_RUNTIME: crun
```

Comment:
why not just set the environment variable right here?

Reply:
Some default value needs to be set globally, then overriden as needed in tasks.  Otherwise Cirrus assumes the value used in the task-name is (literally) "none".